examples/next-fireworks/README.md

```diff
@@ -0,0 +1,42 @@
+# Vercel AI SDK, Next.js, and Fireworks.ai Chat Example
+
+This example shows how to use the [Vercel AI SDK](https://sdk.vercel.ai/docs) with [Next.js](https://nextjs.org/) and [Fireworks.ai](https://app.fireworks.ai) to create a ChatGPT-like AI-powered streaming chat bot. Fireworks.ai's APIs are compatible with OpenAI's so we use the OpenAI JS SDK but change its base URL to point to Fireworks's API with an environment variable.
+
+## Deploy your own
+
+Deploy the example using [Vercel](https://vercel.com?utm_source=github&utm_medium=readme&utm_campaign=ai-sdk-example):
+
+[![Deploy with Vercel](https://vercel.com/button)](https://vercel.com/new/clone?repository-url=https%3A%2F%2Fgithub.com%2Fvercel%2Fai%2Ftree%2Fmain%2Fexamples%2Fnext-fireworks&env=OPENAI_API_KEY&envDescription=Fireworks%20API%20Key&envLink=https%3A%2F%2Fplatform.openai.com%2Faccount%2Fapi-keys&project-name=vercel-ai-chat-fireworks&repository-name=vercel-ai-chat-fireworks)
```

Comment:
@steven-tey can you get ensure this link works after we merge? We need to add another env var for `OPENAI_API_BASE` with value `"https://api.fireworks.ai/inference/v1"`